translations/Marathi/general-rules.md

```diff
@@ -0,0 +1,40 @@
+# General Rules for Translation in Marathi
+
+## Overview
+
+* Marathi is a very versatile & easy-to-use language like English, so translations
+  are also easy.
+
+* Translations can be done in either way, formal or informal, but formal way is preferred
+  for corporate work.
+
+* The translations can be done in any person( first, second or third) according to the 
+  content one wants to translate.
+
+* Same is applicable for singular & plural, the translations depend upon the type of content.
+
+* The usage of verbs & nouns is similar to English, the type of the verb & the noun used changes
+  according to the construction of the sentence & the content, so there are no such rules 
+  for verbs & nouns.
+
+* The usage of particles( e.g. reflexives) again depends upon the sentence, so their is no 
+  restriction on their usage.
+
+* Slang words( especially abusive words) should not be used while translations.
+
+* Long Words are very uncommon in Marathi( especially Zulip-specific), so the translations
+  are easy & quick.
+* There is no specific way of addressing people in Marathi, it can be done in any way( formal 
+  or informal), but Formal way is suggested for Zulip.
+  ( Also, personal pronouns can be used freely, there is no restriction or convention.) 
+
+* Capitalization is not done in Marathi in any form, as there are no capital & small letters 
+  in the language.
+  
+* Other grammatical elements( such as gerunds) are used again according to the content,
+  just like it is done in English.
+
+* Translation of Zulip-specific terms & common verbs is also very easy in Marathi.
+  e.g,
+  1. The Word 'topic' is called 'विषय' in Marathi.
```

Comment:
Can you show some examples of the the type of sentence structure you would recommend? Single word translations are not really a very good example for someone who might be translating and looking for a pattern.